.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Debug Streamlit App",
+      "type": "debugpy",
```

Comment:
is this built in to vscode or does it require an extension of some sort?

Reply:
<img width="306" alt="Screenshot 2024-09-18 at 12 58 16 PM" src="https://github.com/user-attachments/assets/61d73f86-df3c-4005-9f60-6d3c0768237d">

Good question! It requires the user to install an extension. This is what shows up when you first run it without the extension installed. Note that this is the official Microsoft Python Debugger

Reply:
perfect!